rules.rules.events.yml

```diff
@@ -33,3 +33,5 @@ rules_entity_update:
 rules_entity_view:
   label: 'Viewing an entity'
   deriver: '\Drupal\rules\Plugin\RulesEvent\EntityViewDeriver'
+rules_system_logger_event:
+  label: 'System log entry is created'
```

Comment:
So the context of this event is missing? We have a log entry here, do we have a typed data structure for that?